example/windows/ReactNativePosToolsExample/MainPage.xaml

```diff
@@ -10,7 +10,7 @@
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
     <react:ReactRootView 
         x:Name="ReactRootView"
-        ComponentName=""
+        ComponentName="ReactNativePosToolsExample"
```

Comment:
This was the big unlock!

<img width="1057" alt="Screenshot 2024-03-04 at 10 15 40 AM" src="https://github.com/infinitered/react-native-pos-tools/assets/37849890/baa15a1a-c10f-4c98-8f24-b6e63f2aec33">